### PR TITLE
TC19_View-&-Cart-Brand-Products

### PR DESCRIPTION
**Pull request by HebaAbdelaziz911**

### CODE REVIEW

**Review by MahmoudElSharkawyGS — CHANGES_REQUESTED** [review]
- Where's the execution pipeline link ?

File: src/main/java/com/gizasystems/automationexercise/pages/BrandProductsPage.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+public class BrandProductsPage {
+    private final SHAFT.GUI.WebDriver driver;
+    private final By productsbtn_button = By.xpath("//a[@href='/products']");
+    private final By brandsBody = By.xpath("//div[@class='brands_products']");
+    private final By brandNameBtn_button = By.xpath("//a[@href='/brand_products/Polo']");
+    private final By brandName = By.xpath("//div/h2[@class='title text-center']['Brand - Polo Products']");
+    private final By productsOfBrand = By.xpath("//a[@href='/product_details/1']");
+    private final By secondBrandName_button = By.xpath("//a[@href='/brand_products/H&M']");
+    private final By secondBrandName = By.xpath("//div/h2[@class='title text-center']['Brand - H&M Products']");
```
Comment (MahmoudElSharkawyGS) [inline src/main/java/com/gizasystems/automationexercise/pages/BrandProductsPage.java:15]: Brand name should be test data not part of the xpath to be more dynamic.

**Review by MahmoudElSharkawyGS — CHANGES_REQUESTED** [review]
- Please make sure to have a test data json file with the test cases test data.
- Please don't forget to execute the pipeline and share the execution link.

File: src/main/java/com/gizasystems/automationexercise/pages/BrandProductsPage.java (hunk 1 above)
Comment (HebaAbdelaziz911) [reply src/main/java/com/gizasystems/automationexercise/pages/BrandProductsPage.java:15]: Solved.

**Comment by HebaAbdelaziz911** [comment]
The pipeline link:
https://github.com/MahmoudElSharkawyGS/AutomationExercisePractice/actions/runs/8097783320

**Review by MahmoudElSharkawyGS — APPROVED** [review]

**Pull request merged**